index.html

```diff
@@ -20,220 +16,142 @@
       referrerpolicy="no-referrer"
     />
   </head>
-  <body>
-    <!-- navbar -->
-    <div class="navbar">
-      <div class="navbar__s1">
-        <i class="fa-solid fa-bars" style="color: #000000"></i>
+  <body class="font-sans">
+    <!-- Navbar -->
+    <div class="sticky top-0 z-50 flex items-center bg-white p-3">
+      <div class="block lg:hidden">
+        <i class="fa-solid fa-bars text-black"></i>
       </div>
-      <div class="navbar__s2">
-        <img
-          src="./images/Tripadvisor_lockup_horizontal_secondary_registered.svg"
-          alt="Tripadvisor_logo"
-          width="200px"
-        />
+      <div class="flex items-start w-1/5 p-3">
+        <img src="./images/Tripadvisor_lockup_horizontal_secondary_registered.svg" alt="Tripadvisor_logo" class="w-48" />
```

Comment:
logo size should reduce in small screen and it should be center
![image](https://github.com/user-attachments/assets/e2c92195-6882-475a-8eb0-cfb7e7c62dce)
refer this image